docs/en_US/Compression/Overview.rst

```diff
@@ -26,7 +35,7 @@ The algorithms include pruning algorithms and quantization algorithms.
 Pruning Algorithms
 ^^^^^^^^^^^^^^^^^^
 
-Pruning algorithms compress the original network by removing redundant weights or channels of layers, which can reduce model complexity and address the over-ﬁtting issue. 
+Pruning algorithms compress the original network by removing redundant weights or channels of layers, which can reduce model complexity and address the over-ﬁtting issue.
```

Comment:
Address is too strong, how about `mitigate`?

Reply:
Thanks for your suggestion. Have changed to `mitigate`, please review the latest version.